ddcz/templates/users/detail.html

```diff
@@ -1,12 +1,139 @@
 {% extends 'public.html' %}
 {% load html %}
+{% load creations %}
 {% block content %}
+
 <div id="page-user-detail">
     <h1 class="page-heading">Uživatel{% if profile.is_female %}ka{% endif %} {{ profile.nick_uzivatele }}</h1>
     {% if profile.icon_url %}<p><img src="{{ profile.icon_url }}" alt="Ikonka uživatele"></p>{% endif %}
-    {% if profile.is_author %}
-        <p>Uživatel je autorem. <a href="{{ profile.author_url }}">Přejít na autorský profil</a></p>
-    {% endif %}
-    <p><span class="label">Popis:</span> <span class="value">{{ profile.popis_uzivatele }}</span></p>
+
+
+    <div id="user_container">
+        <section id="user_info">
+            <h3>Info o uživatel{% if profile.is_female %}ce{%else%}i{% endif %}</h3>
+            <table>
+                <tr>
+                    <th>ID</th>
```

Comment:
🤔 ...are we going to display it? :)

Reply:
Of course :D Users will know it anyway from the url address and as I remember, people were happy to know their ID.

Reply:
:trollface: